torch/onnx/symbolic_helper.py

```diff
@@ -1598,11 +1577,14 @@ def quantize_helper(
         )
 
     assert scale is not None
-    if scale.type().scalarType() != "Float":
+    if _type_utils.JitScalarType.from_value(scale).scalar_name() != "Float":
         scale = g.op("Cast", scale, to_i=_C_onnx.TensorProtoDataType.FLOAT)
 
     assert zero_point is not None
-    if zero_point.type().scalarType() not in ("Byte", "Char"):
+    if _type_utils.JitScalarType.from_value(zero_point).scalar_name() not in {
+        "Byte",
+        "Char",
+    }:
```

Comment:
Probably leverage the enum here as well?

Reply:
done